src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportResourceLoader.cs

```diff
@@ -151,6 +151,11 @@ private ImportResource ParseImportRawContent(string resourceType, (string Conten
                 {
                     throw new FormatException("Resource type not match.");
                 }
+
+                if (!string.IsNullOrEmpty(resourceType) && importResource.ResourceWrapper != null && importResource.ResourceWrapper.ResourceTypeName.Equals("SearchParameter", StringComparison.Ordinal))
```

Comment:
We don't need to check `resourceType` as it may be null/empty for import requests that don't specify the resource type (files that have mixed resource types).